### PR TITLE
[plugin.video.lbry] 0.1.3

### PR DESCRIPTION
**Pull request by accumulator**
### Description
Update of plugin.video.lbry from Leia to Matrix

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.lbry/resources/language/strings.pot [hunk 1]
```diff
@@ -0,0 +1,144 @@
+msgid ""
```
Comment (enen92) [inline plugin.video.lbry/resources/language/strings.pot]: this file is incorrect, remove it

File: plugin.video.lbry/resources/lib/local.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+import xbmc
+import xbmcaddon
+import xbmcvfs
+import xbmcgui
+
+ADDON = xbmcaddon.Addon()
+tr = ADDON.getLocalizedString
+
+def get_profile_path(rpath):
+    return xbmc.translatePath(ADDON.getAddonInfo('profile') + rpath)
```
Comment (enen92) [inline plugin.video.lbry/resources/lib/local.py]:
```suggestion
    return xbmcvfs.translatePath(ADDON.getAddonInfo('profile') + rpath)
```

File: plugin.video.lbry/resources/lib/exception.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
```
Comment (enen92) [inline plugin.video.lbry/resources/lib/exception.py:1]: empty/dummy `__init__.py` files should no longer be needed in python3

File: plugin.video.lbry/resources/lib/local.py [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    for line in lines.split('\n'):
+        items = line.split('#')
+        if len(items) < 2:
+            continue
+        channels.append((items[0],items[1]))
+    return channels
+
+def save_channel_subs(channels):
+    try:
+        f = xbmcvfs.File(get_profile_path('channel_subs'), 'w')
```
Comment (enen92) [inline plugin.video.lbry/resources/lib/local.py]: jfyi `xbmcvfs.File` does support context lib, so you should be able to use:

```python
with xbmcvfs.File(get_profile_path('channel_subs'), 'w') as f:
    f.something()
``` 

Not mandatory to change though.

**Review by enen92 — CHANGES_REQUESTED** [review]
See comments

**Comment by accumulator** [comment]
@enen92 issues resolved

**Review by enen92 — APPROVED** [review]

**Pull request merged**